.github/workflows/e2etests.yaml

```diff
@@ -41,10 +41,11 @@ jobs:
 
   TestCilium:
     needs: SetPatternfile
-    uses: meshery/meshery/.github/workflows/testadapters.yaml@master
+    uses: meshery/meshery/.github/workflows/test_adapters.yaml@master
     with:
-      expected_pods: cilium
```

Comment:
I wonder why there is only 1 expected pod but 3 namespaces? // @MUzairS15

Reply:
@gr455 In actual cilium has 3 expected pods to come up  but due to some issue the other pods tend to get killed, so I removed from expected pods, but forgot to remove namespaces 😅.

I will address this by creating another PR.

Reply:
Ah. Okay then